Lib/fontParts/base/point.py

```diff
@@ -260,12 +263,12 @@ def _set_x(self, value):
         """
     )
 
-    def _get_base_y(self):
+    def _get_base_y(self) -> Number:
         value = self._get_y()
         value = normalizers.normalizeY(value)
         return value
 
-    def _set_base_y(self, value):
+    def _set_base_y(self, value: Number):
```

Comment:
Should return `None`. `value` should be `annotations.IntFloatType`, not `Number`